Formula/percona-server.rb

```diff
@@ -42,16 +48,11 @@ def datadir
   end
 
   def install
-    # Set HAVE_MEMSET_S flag to fix compilation
-    # https://bugs.launchpad.net/percona-server/+bug/1741647
-    ENV.prepend "CPPFLAGS", "-DHAVE_MEMSET_S=1"
-
-    # https://dev.mysql.com/doc/refman/5.7/en/source-configuration-options.html
     # -DINSTALL_* are relative to `CMAKE_INSTALL_PREFIX` (`prefix`)
     args = %W[
       -DCOMPILATION_COMMENT=Homebrew
-      -DDEFAULT_CHARSET=utf8
-      -DDEFAULT_COLLATION=utf8_general_ci
+      -DDEFAULT_CHARSET=utf8mb4
+      -DDEFAULT_COLLATION=utf8mb4_general_ci
```

Comment:
Is `utf8mb4_0900_ai_ci` much better?

Reply:
@mbcoguno That is the default collation of MySQL 8.0, but current mysql formula doesn't. https://github.com/Homebrew/homebrew-core/commit/523589bf5b03053e40fb28681a5eac42655ce4cc#diff-4d75be177e3055025e7bef3144311559R49

Reply:
We want to ship with the most common configuration.